src/cls/2pc_queue/cls_2pc_queue_types.h

```diff
@@ -10,23 +10,26 @@ struct cls_2pc_reservation
   inline static const id_t NO_ID{0};
   uint64_t size;        // how many entries are reserved
   ceph::coarse_real_time timestamp;  // when the reservation was done (used for cleaning stale reservations)
+  uint32_t entries;
 
-  cls_2pc_reservation(uint64_t _size, ceph::coarse_real_time _timestamp) :
-      size(_size), timestamp(_timestamp) {}
+  cls_2pc_reservation(uint64_t _size, ceph::coarse_real_time _timestamp, uint32_t _entries) :
+      size(_size), timestamp(_timestamp), entries(_entries) {}
 
   cls_2pc_reservation() = default;
 
   void encode(ceph::buffer::list& bl) const {
-    ENCODE_START(1, 1, bl);
+    ENCODE_START(2, 1, bl);
     encode(size, bl);
     encode(timestamp, bl);
+    encode(entries, bl);
     ENCODE_FINISH(bl);
   }
 
   void decode(ceph::buffer::list::const_iterator& bl) {
-    DECODE_START(1, bl);
+    DECODE_START(2, bl);
     decode(size, bl);
     decode(timestamp, bl);
+    decode(entries, bl);
```

Comment:
you have to protect this with `struct_v >= 2` for the case you read the old format